libs/velo-external-db-core/src/router.ts

```diff
@@ -122,12 +122,12 @@ export const createRouter = () => {
 
     router.post('/provision', async(req, res) => {
         const { type, vendor } = cfg
-        res.json({ type, vendor, protocolVersion: 3 })
+        res.json({ type, vendor, protocolVersion: 3, adapterVersion: 'v1' })
```

Comment:
What use do you have with `adapterVersion`?
Should we pass hard coded 'v1' or it should change if we're updating the adapter?

Reply:
It's only used in BI events. It should indicate the version of adapter code - so it could be a docker image version, or npm version.

This property has been around in v1 and v2 connectors, but I don't see it being filled... Now in v3 we marked it as mandatory.

Here's the BI event:
https://bo.wix.com/bi-catalog-webapp/#/sources/83/events/262/data-samples?artifactId=com.wixpress.cloud.cloud-data